doczrc.js

```diff
@@ -50,15 +50,34 @@ const projectPlugin = () =>
     },
   });
 
+/**
+ * Atlantis contains some private components. These are indicated by an
```

Comment:
What does this function do? That should be the first part of the comment. The other notes are nice (and can be included) but don't explain the why of the function.